src/components/Dropdown/PrioritySelector/PrioritySelector.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
```

Comment:
I would place it in its own, as an organism. I think inside other components mean, that they are used by that component, not that they use it. Then we can also change implementation details in the future if needed (I think you saw that design...)

Reply:
In the refinement session we decided that it was going in the dropdown component's folder... I do agree though.
Shall i change it to it's own folder, and change the ticket?